src/apify_client/clients/resource_clients/webhook_dispatch_collection.py

```diff
@@ -0,0 +1,26 @@
+from typing import Any, Dict, Optional
+
+from ..base.resource_collection_client import ResourceCollectionClient
+
+
+class WebhookDispatchCollectionClient(ResourceCollectionClient):
+    """Sub-client for listing webhook dispatches."""
+
+    def __init__(self, *args: Any, **kwargs: Any) -> None:
+        """Initialize the WebhookDispatchCollectionClient."""
+        resource_path = kwargs.pop('resource_path', 'webhook-dispatches')
```

Comment:
Why is the `pop` needed here? Can't you simply hard code `webhook-dispatches` on the next line in `super().__init__(...)`? That's how it's done in e.g. dataset collection client.

Reply:
You can't, because you have to get resource_path out from the dictionary if it is there, otherwise it will lead to an error because you would be passing one named argument twice.

Reply:
can you please leave a comment here? Thanks